addons/website_sale/controllers/main.py

```diff
@@ -1227,3 +1227,50 @@ def get_combination_info_website(self, product_template_id, product_id, combinat
                 'product_variant': request.env['product.product'].browse(res['product_id']),
             }))
         return res
+
+    # ------------------------------------------------------
+    # Products Search Bar
+    # ------------------------------------------------------
+
+    def get_product_searchbar_domain(self, term, product_options, **kwargs):  # set to use old function with cat and attr
```

Comment:
It seems that your work adds python code but we already have a search bar in /shop which work, should there not be some code shared ? I don't know, just want you to check during your review. JKE will probably be better to say if it is ok.

Reply:
I tried to use _get_search_domain() which already exists but that's not exactly what I need as I let the user searching or not in the product description. But still, idk if in that case it's better to adapt or not.